Resources/config/services.xml

```diff
@@ -11,15 +11,15 @@
     </parameters>
 
     <services>
-        <service id="knp_disqus.request" class="%knp_disqus.api.class%">
+        <service id="knp_disqus.request" class="%knp_disqus.api.class%" public="true">
             <argument type="service" id="service_container" />
             <argument>%knp_disqus.api_key%</argument>
             <argument>%knp_disqus.secret_key%</argument>
             <argument>%knp_disqus.base_url%</argument>
             <argument>%kernel.debug%</argument>
         </service>
 
-        <service id="knp_disqus.helper" class="%knp_disqus.helper.class%">
+        <service id="knp_disqus.helper" class="%knp_disqus.helper.class%" public="true">
```

Comment:
I don’t think this service needs to be public

Reply:
helper class is used by twig function "knp_disqus_render"

Reply:
You’re right :/